OneLogin.xcodeproj/project.pbxproj

```diff
@@ -2055,11 +2060,11 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_TEAM = N8W395F695;
+				EXTERNAL_BASE_URL = signin.account.gov.uk;
 				FEATURE_FLAG_FILE = FeatureFlagsBuild;
 				GENERATE_INFOPLIST_FILE = YES;
 				INFOPLIST_FILE = "OneLogin-Info.plist";
-				INFOPLIST_KEY_CFBundleDisplayName = "One Login - Build";
-				INFOPLIST_KEY_NSCameraUsageDescription = "This app does not use the camera";
```

Comment:
Looks like the Xcode project file is doing some annoying stuff here in terms of renaming the app display name and getting rid of the camera usage description. I've seen this happen locally as well, very annoying.
Worth keeping an eye out for changes in this project file, we've fallen foul of stuff like this on the program in the past with stuff making it's way into production... less of a concern for this app atm but good practice to get into 🙂
With bringing these project settings back, there's a nice way of toggling settings names/titles in Xcode (if you don't know about it already).
When you're in the project file, Editor -> Show Setting Names will show them in the format you see here and may make it a little easier
![image 2024-04-22 at 14 31 56](https://github.com/govuk-one-login/mobile-ios-one-login-app/assets/90717175/b2956545-4e1b-4c97-b7e3-4e45dc91dd03)

Reply:
Is the only way to change these to revert them manually to what they are in the commit changes? And there's no way to stop Xcode randomly doing this?